scripts/TestHarness/testers/RavenUtils.py

```diff
@@ -53,6 +53,8 @@ def __lookUpPreferredVersion(name):
                ("scipy",__lookUpPreferredVersion("scipy")),
                ("scikit-learn",__lookUpPreferredVersion("sklearn")),
                ("matplotlib",__lookUpPreferredVersion("matplotlib")),
+               ("xarray",""),
+               ("netcdf4",""),
```

Comment:
do we have a preferred version?

Reply:
We will, but not yet.  These will be added when we add tensorflow and determine versions for the remaining libraries.

Reply:
ok, this PR is good to me.